.changeset/eleven-nights-relate.md

```diff
@@ -0,0 +1,7 @@
+---
+'mastra': patch
+'@mastra/core': patch
+'create-mastra': patch
```

Comment:
Shouldn't this just be the playground package?

Reply:
playground is included into core and create-mastra, I thought I should include it, maybe I should not :)